plugins/odf/console-extensions.json

```
@@ -747,7 +747,7 @@
       }
     },
     "flags": {
-      "required": ["MCG_STANDALONE"]
+      "required": ["MCG"]
```

it's for MCG standalone deployments only !
```suggestion
      "required": ["MCG_STANDALONE"]
```